platform/kube/ingressstatus.go

```diff
@@ -57,11 +59,16 @@ func NewIngressStatusSyncer(mesh *config.ProxyMeshConfig, client *Client,
 		&v1beta1.Ingress{}, options.ResyncPeriod, cache.Indexers{},
 	)
 
+	var publishService string
+	if mesh.IngressService != "" {
```

Comment:
i think it's better not to run status syncer when the field is not defined, WDYT?